lib/ansible/modules/apt_repository.py

```diff
@@ -78,6 +78,16 @@
               a non-Ubuntu target (for example, Debian or Mint).
         type: str
         version_added: '2.3'
+    install_python_apt:
+        description:
+            - Whether to automatically try to install Python apt or not, if it is not already installed.
+            - If python-apt Python library is not installed this module will not work.
+            - Note that it is installed by running C(apt-get install python-apt) for Python 2, and C(apt-get install python3-apt) for Python 3.
+              If you are using a Python on the remote that is not the system Python, this will not work. In that case,
+              you should set I(install_python_apt=false) and need to ensure that the Python apt module for your
+              Python version is installed in another way.
+        type: bool
+        default: true
```

Comment:
This probably needs some rewording.  I'm also curious as to when it was added.

I'll leave this open for review from the docs team.

Reply:
The original option was added here over 6.5 years ago: https://github.com/ansible/ansible-modules-core/commit/5c021fec3ce19212351a449d28d4c26f0c3580c0